services/movie_session.py

```diff
@@ -42,3 +42,12 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> Ticket:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+
+    taken_seats = [{"row": ticket.row, "seat": ticket.seat} for ticket in
+                   tickets]
+
+    return taken_seats
```

Comment:
The `get_taken_seats` function should return a list of dictionaries, not a `Ticket` object. Ensure the return type in the function signature matches the actual return value, which is a list of dictionaries representing taken seats.